service/src/main/java/io/camunda/service/MessageServices.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service;
+
+import io.camunda.search.clients.CamundaSearchClient;
+import io.camunda.service.security.auth.Authentication;
+import io.camunda.service.transformers.ServiceTransformers;
+import io.camunda.zeebe.broker.client.api.BrokerClient;
+import io.camunda.zeebe.gateway.impl.broker.request.BrokerCorrelateMessageRequest;
+import io.camunda.zeebe.protocol.impl.record.value.message.MessageCorrelationRecord;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public final class MessageServices<T> extends ApiServices<MessageServices<T>> {
```

Comment:
❓ Do we need the generic here for anything specific? Looking at the [IncidentServices](https://github.com/camunda/camunda/blob/main/service/src/main/java/io/camunda/service/IncidentServices.java), for example, we don't usually need this. The JobServices need the generic because the type of the job activation handler is generic as well. But most (if any) other services should not need this except for special cases. I don't see any reason for the `UserServices` to have it, for example. But maybe I am missing something here 🤔

Reply:
Probably not actually. I took inspiration from the JobServices so I figured it was just how things were done 😄 I'll remove it.

Reply:
I'll do it for the UserServices as well while I'm at it as I let it pass review 👍